docker/standup_base

```diff
@@ -26,6 +26,4 @@ COPY requirements.txt /app/requirements.txt
 RUN pip3 install --require-hashes --no-cache-dir -r requirements.txt
 
 COPY . /app
-
-RUN adduser --uid 1000 --disabled-password --gecos '' --no-create-home webdev
-RUN chown webdev.webdev -R .
+RUN DEBUG=False SECRET_KEY=foo DATABASE_URL=sqlite:// ./manage.py collectstatic --noinput
```

Comment:
I like that. I don't like the ugly command in the docker file, but at build time there are no env vars, and doing the collectstatic step at run time in `run-prod.sh` isn't optimal since that delays the starting of the server (which will likely get worse if we have a sass processing or minification step). I can't immediately think of another management command we'd need to run, but it won't hurt to have the option. I'll add this. Thanks 🍰